enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/VolumePermissionEnricher.java

```diff
@@ -50,7 +51,8 @@
     private final InitContainerHandler initContainerHandler;
 
     enum Config implements Configs.Key {
-        permission {{ d = "777"; }};
+        permission {{ d = "777"; }},
+        storageClass {{ d = ""; }};
```

Comment:
Shouldn't we simply omit a default value and when empty then just not add a storage class ?